test/LockedNORI.test.ts

```diff
@@ -698,36 +733,60 @@ describe('LockedNori', () => {
 
   it('Should unlock the full grant at endtime', async () => {
     // now == endTime
-    const startTime = NOW;
-    const { lNori } = await setupWithGrant({ startTime });
+    const { nori, lNori, grantAmount, grant } = await setupWithGrant();
     const { investor1 } = await hre.getNamedAccounts();
     const addr1Signer = await hre.ethers.getSigner(investor1);
     await hardhat.network.provider.send('evm_setNextBlockTimestamp', [
-      NOW + END_OFFSET,
+      grant.unlockEndTime,
```

Comment:
Thanks for changing usage of constants. This was a big item on my to-do list as I found it difficult to expect values previously when wanting to test alternative schedules